### PR TITLE
Fixes for 1.4.0 patch

### PR DESCRIPTION
**Pull request by tony56a**
Seems like CO made some changes internally to the UI stuff that broke the mod. Updated it reflect new methods, and it seems like it's working again.

### CODE REVIEW

File: FineRoadHeights/BeautificationFinePanel.cs [hunk 1]
```diff
@@ -50,10 +47,7 @@ protected override void OnButtonClicked(UIComponent comp)
             NetToolFine netTool = ToolsModifierControl.SetTool<NetToolFine>();
             if (netTool != null)
             {
-                if (base.pathsOptionPanel != null)
-                {
-                    base.pathsOptionPanel.Show();
-                }
+                this.HidePathsOptionPanel();
```
Comment (the-m4a) [inline FineRoadHeights/BeautificationFinePanel.cs]: Should this be ShowPathsOptionPanel?

File: FineRoadHeights/PublicTransportFinePanel.cs [hunk 2]
```diff
@@ -77,12 +71,9 @@ protected override void OnButtonClicked(UIComponent comp)
     if (!((UnityEngine.Object) netTool != (UnityEngine.Object) null))
       return;
     ItemClass.SubService subService = netInfo.GetSubService();
```
Comment (the-m4a) [inline FineRoadHeights/PublicTransportFinePanel.cs:73]: (new line 73) this line is also safe to remove (not using the subService object anywhere)

**Comment by RosaryMala** [comment]
Considering that 1.4.0 partially implements the functionality of this mod, I'm about 90% sure more than just this needs changing.

**Comment by the-m4a** [comment]
True, though this mod offers better granularity and the very desirable ability to cycle between ground, normal, elevated, and bridge.
FWIW, I pulled down a copy of the source and made the changes here and it wasn't perfect, which I don't know if is a result of caching and not updating like I thought or what.  For example, I didn't have the road snap or geometry (straight, curved, etc) buttons, and highway seemed to treat normal as elevated.  It got too late at night to investigate any further :/

**Comment by tony56a** [comment]
Sorry about that, main concern for me was getting it to build+ getting the road height to show up. I think there's some UI issues( it's now hiding some of the new road UI buttons), plus the ground/ elevated/ bridge toggling

**Comment by the-m4a** [comment]
Hah, nothing to be sorry for - I was appreciative someone had looked into the issues on day 0, as much as I am appreciative of it being on GitHub in the first place.  (this is the part where I lament the lack of time to look into this some more, sooner than later)

**Pull request closed without merging**